readme.md

```diff
@@ -1 +1,60 @@
-# Hello World
+# Form Builder
+
+**Form Builder** is a simple node application that generates a form based on given inputs.
+
+<div align="center">
```

Comment:
Alignment should be excluded from markdown... as well as HTML, yes it renders, but this is an invalid markdown file.